test/Microsoft.DotNet.SourceBuild.Tests/assets/LicenseScanTests/Licenses.runtime.json

```diff
@@ -1,5 +1,9 @@
 {
   "files": [
+    {
+      "path": "THIRD-PARTY-NOTICES.TXT",
+      "detected_license_expression": "apache-2.0 AND bsd-new AND unicode AND zlib AND mit AND iso-8879 AND bsd-simplified AND w3c-software-doc-20150513 AND public-domain AND ietf AND osf-1990 AND uoi-ncsa AND cc0-1.0 AND warranty-disclaimer AND bsd-original-uc AND (mit AND object-form-exception-to-mit) AND nist-software"
+    },
```

Comment:
Related to https://github.com/dotnet/dotnet/pull/224/files#diff-ea0539428c8059d933167d1f75966dffa2b32d202fc9849c6adff596cb94980c